app/client/components/SpeedIndicator.js

```diff
@@ -0,0 +1,39 @@
+
+import React from 'react';
+
+export default class SpeedIndicator extends React.Component {
+
+    static propTypes = {
+        speed: React.PropTypes.number.isRequired,
+    }
+
+    static defaultProps = {
+        speed: 'wer',
```

Comment:
you have defined the property `speed` as a _required_ numeric input so:

1. you don't need a default value 'cause is required
2. even if you default it you should default to a number